README.md

```diff
@@ -84,6 +90,8 @@ This custom View folder can be downloaded from your Primo Back Office, by follow
 
       - [CSS](https://github.com/ExLibrisGroup/primo-explore-package/tree/master/VIEW_CODE/css "css documentation")
 
+      - [SASS/SCSS](https://github.com/ExLibrisGroup/primo-explore-package/tree/master/VIEW_CODE/scss "sass/scss documentation") (an optional alternative to CSS)
```

Comment:
These links won't work yet since the https://github.com/ExLibrisGroup/primo-explore-package needs to be updated accordingly